optimum/exporters/tasks.py

```diff
@@ -636,6 +636,10 @@ class TasksManager:
             "sequence-classification",
             onnx="PerceiverOnnxConfig",
         ),
+        "pix2struct": supported_tasks_mapping(
+            "default",
```

Comment:
you should probably remove the `default` here (AutoModel can not load pix2struct), and just have `image-to-text`.

The issue is that there is no autoclass in transformers for image-to-text, so we will need to add a way to specify explicitly in tasks.py that for pix2struct the class to use is Pix2StructForConditionalGeneration. Let me do a PR for this.

Reply:
done